draft-yasskin-http-origin-signed-responses.md

```diff
@@ -181,13 +181,13 @@ values:
 
```

Comment:
In the non-diff above: "This specification gives no meaning to the identifier itself". Per httpwg/http-extensions#720, you may want to update this to "the primary identifier itself".

Reply:
Hm, https://github.com/httpwg/http-extensions/commit/3c96a9b14e82f0356f09ac680a2943bdca9830fc then removes that term in favor of "token". I think I'm going to keep just "identifier" here, since I think it's clear enough that it refers to the parameterised thing, and not any of the parameters, but @mnot FYI in case you want dependent specs to do something else.